Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -777,7 +777,8 @@ instance : Inhabited (Subalgebra R A) := ⟨⊥⟩
 
 theorem mem_bot {x : A} : x ∈ (⊥ : Subalgebra R A) ↔ x ∈ Set.range (algebraMap R A) := Iff.rfl
 
-theorem toSubmodule_bot : Subalgebra.toSubmodule (⊥ : Subalgebra R A) = 1 := rfl
+theorem toSubmodule_bot : Subalgebra.toSubmodule (⊥ : Subalgebra R A) = 1 :=
```

Comment:
Can you add a TODO comment here that references the same issue, so we remember to turn this back to a rfl?

Reply:
```suggestion
/-- TODO: change proof to `rfl` when fixing #18110. -/
theorem toSubmodule_bot : Subalgebra.toSubmodule (⊥ : Subalgebra R A) = 1 :=
```